Sources/Covfefe/Grammar.swift

```diff
@@ -238,6 +238,10 @@ extension Grammar: CustomStringConvertible {
 						}
 						
 						return "\(lowerString) ... \(upperString)"
+					
+					case .terminal(.characterSet(let aSet, _)):
+						#warning("Do we want to print a Character Set?")
```

Comment:
hmm, BNF has no notion of a character set. It's somewhat easy to express a character range with a slight modification of the BNF/EBNF/ABNF grammar, but I don't see an easy way of expressing a character set (which can potentially be huge).

EBNF has the notion of a special sequence starting and ending with a question mark (`? foo bar ?`). We could use the question mark delimiter here as well.

Note that there's a typo here, should be `character set`, not `chacater set`. (This typo is also in some other places).